import-export-cli/integration/testutils/testConstants.go

```diff
@@ -44,11 +44,16 @@ const DeploymentEnvYamlFilePath = "deployment_environments.yaml"
 const SampleAPIYamlFilePath = "testdata/sample-api.yaml"
 const SampleRevisionedAPIYamlFilePath = "testdata/sample-revisioned-api.yaml"
 const DevFirstUpdatedSampleCaseArtifactPath = "testdata/TestArtifactDirectory/DevFirstUpdatedSampleCaseArtifacts"
+const DevFirstHiddenFiles = "testdata/TestArtifactDirectory/HiddenFiles"
```

Comment:
Since the only usage of this constant is for `DevFirstUpdatedSampleCaseHiddenFilePath` which has no usage,  we can remove this too right?

Reply:
Yeah, will remove unwanted files & constants @ashera96